modules/apps/object/object-api/src/main/java/com/liferay/object/field/setting/util/ObjectFieldSettingUtil.java

```diff
@@ -111,6 +111,19 @@ public static String getValue(String name, ObjectField objectField) {
 		return getValue(name, objectField.getObjectFieldSettings());
 	}
 
+	public static boolean isUnique(ObjectField objectField) {
+		if ((objectField == null) ||
+			(objectField.getObjectFieldSettings() != null)) {
```

Comment:
I think that it should be `if (objectField == null || objectField.getObjectFieldSettings() == null)` because if it is not not null it will always return false. Or am I missing something?

Reply:
You are right!